### ISSUE TITLE
Project maintained?

### ISSUE THREAD

**Issue by pada57** [benaadams/Ben.Demystifier#223]
Hi
I'm interested by this project as solves interesting issue on developer side. Would like to know if you maintain it, if yes do you accept PR as don't see any PR merged yet

Did you create another forked project under different code name ?

Cheers

**Comment by bruno-garcia** [comment]
Sentry [maintains a fork](https://github.com/getsentry/Ben.Demystifier) for our own use. But we're not publishing it as a library by itself, it's bundled in the [Sentry .NET SDK](https://github.com/getsentry/sentry-dotnet)

@benaadams if you'd like help maintaining this, I'd be happy to.

**Comment by moegodot** [comment]
Try https://github.com/mingmoe/Mingmoe.Demystifier , the fork with new features

**Comment by alfonsovgs** [comment]
> Try https://github.com/mingmoe/Mingmoe.Demystifier , the fork with new features

Is it available as a nuget?

**Comment by moegodot** [comment]
> > Try https://github.com/mingmoe/Mingmoe.Demystifier , the fork with new features
> 
> Is it available as a nuget?

Yes. I have update the README.md to show where the package on nuget is.

**Comment by lonix1** [comment]
Is it still useful in the latest version (v8 / v9) ?

**Comment by bruno-garcia** [comment]
> Is it still useful in the latest version (v8 / v9) ?

At Sentry we continue to use it on v8 and v9.
Why do you mean? Did stack trace rendering change OOTB to the level this project improves readability?

**Comment by lonix1** [comment]
Hi @bruno-garcia 

I found this library via https://github.com/nblumhardt/serilog-enrichers-demystify, which was deprecated with the explanation:
> Important note: This package is no longer being actively developed, as the improved stack trace functionality is now a part of .NET Core, and thus the future utility of this integration package is limited.

Also this library's README states:
> Output the modern C# 7.0+ features in stack traces that looks like the C# source code that generated them rather than IL formatted

So I inferred that in v7 the library was no longer needed, but was useful for older versions.

Is my understanding incorrect?

**Comment by Suchiman** [comment]
@lonix1 
> I found this library via https://github.com/nblumhardt/serilog-enrichers-demystify, which was deprecated with the explanation:
> 
> > Important note: This package is no longer being actively developed, as the improved stack trace functionality is now a part of .NET Core, and thus the future utility of this integration package is limited.

There's a replacement package btw. https://github.com/Megasware128/serilog-enrichers-demystifier

And while there's some improvements to the builtin stacktraces, it's nowhere near what Ben.Demystifier does

**Comment by bruno-garcia** [comment]
> Hi [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/u/1633368?u=856a9e52d4cdc92eac2ff9e1f97900df39ae5e4d&amp;v=4&amp;size=80">@bruno-garcia](https://github.com/bruno-garcia)
> 
> I found this library via [nblumhardt/serilog-enrichers-demystify](https://github.com/nblumhardt/serilog-enrichers-demystify), which was deprecated with the explanation:
> 
> > Important note: This package is no longer being actively developed, as the improved stack trace functionality is now a part of .NET Core, and thus the future utility of this integration package is limited.
> 
> Also this library's README states:
> 
> > Output the modern C# 7.0+ features in stack traces that looks like the C# source code that generated them rather than IL formatted
> 
> So I inferred that in v7 the library was no longer needed, but was useful for older versions.
> 
> Is my understanding incorrect?

There has been a lot of improvements in .NET itself, but still, as @Suchiman said, it's nowhere near what this library can do.

**Comment by TFTomSun** [comment]
> > > Try https://github.com/mingmoe/Mingmoe.Demystifier , the fork with new features
> > 
> > 
> > Is it available as a nuget?
> 
> Yes. I have update the README.md to show where the package on nuget is.

The repository doesn't allow to open an issue. I'd like to get the target framework problem solved by adding explicit target framework support for the currently maintained target frameworks (net8.0 and net9.0) https://github.com/benaadams/Ben.Demystifier/issues/205

**Comment by moegodot** [comment]
@TFTomSun I changed the settings,issues was enabled,go ahead.
